Libraries/WebSocket/WebSocket.js

```diff
@@ -203,7 +202,7 @@ class WebSocket extends EventTarget(...WEBSOCKET_EVENTS) {
   }
 
   _close(code?: number, reason?: string): void {
-    if (Platform.OS === 'android') {
+    if (WebSocketModule.close.length > 1) {
```

Comment:
How about `WebSocketModule.close.length === 3`?

Reply:
Works for me. Coming right up.